plugins/alarmclock/src/test/java/de/unistuttgart/iaas/amyassist/amy/plugin/alarmclock/AlarmClockResourceTest.java

```diff
@@ -171,7 +167,8 @@ void testNewAlarm() {
 		assertEquals(200, r.getStatus());
 		Timestamp tsr = r.readEntity(Timestamp.class);
 		assertEquals(new Timestamp(newAlarm), tsr);
-		assertEquals(Server.BASE_URI.toString() + "/" + AlarmClockResource.PATH + "/alarms/17", tsr.getLink().toString());
+		assertEquals(this.target.getUriBuilder().path(AlarmClockResource.class, "getAlarm").build(17).toString(),
+				tsr.getLink().toString());
```

Comment:
remove both `.toString()` calls

Reply:
Did not really introduce that, but a merge from dev, but will fix anyway.